saml/www/login.html

```diff
@@ -88,25 +88,13 @@ <h4>{{ _("Javascript is disabled on your browser") }}</h4>
 		{{ logo_section() }}
 		<div class="login-content page-card">
 			<form class="form-signin form-login" role="form">
-				{%- if social_login or login_with_email_link -%}
+				{%- if social_login or login_with_email_link or saml_login -%}
 				<div class="page-card-body">
 					<form class="form-signin form-login" role="form">{{ email_login_body() }}</form>
 					<div class="social-logins text-center">
 						{% if not disable_user_pass_login or (ldap_settings and ldap_settings.enabled) %}
 						<p class="text-muted login-divider">{{ _("or") }}</p>
 						{% endif %}
-						<div class="social-login-buttons">
```

Comment:
This isn't necessary, `saml_logins` isn't in context, the SAML providers is added to `provider_logins ` context variable